people/sources.json

```diff
@@ -23,6 +23,7 @@
   "jakub",
   "kuba",
   "pavel",
-  "ivana"
+  "ivana",
+  "eliska"
```

Comment:
tady ted key musi sedet na ten key v tom jsonu, jeden ma hacek, druhy nema to ti nebude fungovat

Reply:
opraveno